core/src/test/scala/unit/kafka/network/SocketServerTest.scala

```diff
@@ -21,24 +21,26 @@ package kafka.network;
 import java.net._
 import javax.net.ssl._
 import java.io._
+import java.util
```

Comment:
Sorry, rather, we should import `java.util.HashMap` rather than importing `java.util` and referring to it as `util.HashMap`.